packages/ui/src/Common.ts

```diff
@@ -509,28 +509,12 @@ export type TooltipDirection = "top" | "bottom" | "left" | "right";
 
 export type IndicatorDirection = "topLeft" | "topRight" | "bottomLeft" | "bottomRight";
 
-type BaseSegmentedControlProps = {
+export interface SegmentedControlProps {
   items: string[];
```

Comment:
does items need to be added to the alphabetized props doc?

Reply:
Added!